allenact/algorithms/onpolicy_sync/runner.py

```diff
@@ -256,13 +273,14 @@ def start_train(
                     if self.running_validation
                     else None,
                     checkpoints_dir=self.checkpoint_dir(),
-                    seed=seed,
+                    seed=trainer_seeds[trainer_it],
```

Comment:
I think this is going to break. the original (shared) seed is used to initialize the model...